website/plugins.js

```diff
@@ -1,8 +1,9 @@
 module.exports = [
   {
     name: 'AdMob',
-    pub: 'firebase_admob',
+    pub: 'google_mobile_ads',
     firebase: 'admob',
+    remoteSource: 'https://github.com/googleads/googleads-mobile-flutter',
     documentation:'',
```

Comment:
```suggestion
    documentation: 'https://pub.dev/documentation/google_mobile_ads/latest/',
```

As suggested by @rrousselGit